app/src/main/java/org/mate/accessibility/AccessibilityViolationTypes.java

```diff
@@ -3,13 +3,14 @@
 public class AccessibilityViolationTypes {
```

Comment:
Not relevant for this commit, but in general this class should probably be an enum that holds the corresponding name as a value (analogous to https://stackoverflow.com/a/1067369)

Reply:
done